### DISCUSSION TITLE
COD WAW sounds

### DISCUSSION THREAD

**Discussion by TheRebreathing** [nzp-team/nzportable#493] [Other Content Releases]
![sounds](https://github.com/nzp-team/nzportable/assets/152328160/6974c342-aa4c-4ce9-a1e0-9925f3947911)
Replaces default sounds with waw sounds
[Download (PC, Vita)](https://www.mediafire.com/file/crpe86qkkcu6jk1/cod+waw+sounds+for+nzp.zip/file)
[Download (Switch)](https://www.mediafire.com/file/9vh6hnl73jwajxs/cod+waw+sounds+for+nzp+switch.zip/file)
[Download (3DS, PSP,)](https://www.mediafire.com/file/5naegoujes88qxy/cod+waw+sounds+for+nzp+psp.zip/file)

**Comment by borconc** [comment]
it's weird, using the package labeled for switch version doesn't seem to work for me. I keep getting the messages that some sounds are stereo which I guess the game doesn't like. I had to use the PSP package to get it to work on switch. Even with the PSP package installed it seems some sounds are still having errors, a different message will pop up each time a perk is bought as well as using the package punch machine.

**Reply by SgtCookieBobble** [reply]
Not OP, but I think I've narrowed it down - player, Pack-a-Punch, and PPSH bolt release sounds were the only ones that worked on my Vita; everything else required me to download the Switch files and I haven't had any sound issues since then.

**Comment by curtmanps4-gif** [comment]
I do have one issue everything else works other than the Perk jingles does anybody have a fix for that
